hardware/EvohomeWeb.cpp

```diff
@@ -77,7 +77,7 @@ void CEvohomeWeb::Init()
 
 bool CEvohomeWeb::StartSession()
 {
-	_log.Log(LOG_NORM, "EvohomeWeb: connect to Evohome server");
+	_log.Log(LOG_NORM, "EvohomeWeb: start new session with Evohome server");
```

Comment:
24 **planned** log messages yes. A session restart can also be triggered by connection timeouts and login failures. Of course LOG_NORM is the highest verbose level of logging.